### PR TITLE
CORDA-1764: Make blobinspector use picocli base class

### PR DESCRIPTION
**Pull request by anthonykeenan**
Also harmonise the documentation for all of the CLI apps.

### CODE REVIEW

File: docs/source/blob-inspector.rst [hunk 1]
```diff
@@ -90,3 +90,25 @@ Here's what a node-info file from the node's data directory may look like:
...
+    blob-inspector [-hvV] [--full-parties] [--install-shell-extensions] [--schema]
+                   [--format=type] [--input-format=type]
+                   [--logging-level=<loggingLevel>] [SOURCE]
+
+* ``--format=type``: Output format. Possible values: [YAML, JSON]. Default: YAML.
+* ``--input-format=type``: Input format. If the file can't be decoded with the given value it's auto-detected, so you should
+  never normally need to specify this. Possible values [BINARY, HEX, BASE64]. Default: BINARY.
+* ``--full-parties``: Display the owningKey and certPath properties of Party and PartyAndReference objects respectively.
+* ``--schema``: Print the blob's schema first.
+* ``--verbose``, ``--log-to-console``, ``-V``: If set, prints logging to the console as well as to a file.
```
Comment (tlil) [inline docs/source/blob-inspector.rst]: Are both this and version abbreviated to `-V` (capital V)?

File: tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt [hunk 2]
```diff
@@ -68,17 +55,19 @@ class BlobInspector : Runnable {
...
-    var verbose: Boolean = false
+    override fun runProgram() = run(System.out)
 
-    override fun run() = run(System.out)
-
-    fun run(out: PrintStream) {
+    override fun initLogging() {
         if (verbose) {
-            System.setProperty("logLevel", "trace")
+            loggingLevel = Level.TRACE
```
Comment (tlil) [inline tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: In the other utilities, this is setting log-to-console as well, isn't it? Does this make the utility behave differently to the others?

Comment (anthonykeenan) [reply tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: Yes it does, it has a custom `log4j2.xml` file which only logs to console. I can change it to use the shared one like I did with the bootstrapper. Wasn't sure about making too many changes.

Comment (anthonykeenan) [reply tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: Didn't want to incur the Wrath of Kat, my favourite star trek fim. @fenryka

File: docs/source/blob-inspector.rst (hunk 1 above)
Comment (anthonykeenan) [reply docs/source/blob-inspector.rst]: I've fixed it, it was wrong for all of the docs.

File: tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt (hunk 2 above)
Comment (tlil) [reply tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: Cool, will leave it to Kat :-)

Comment (fenryka) [reply tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: My argument here is that the blob inspector is a command line utility, not a server / service. I would expect to run it and have it display expected things... logging is going to get in the way of that 99% of the time

Comment (anthonykeenan) [reply tools/blobinspector/src/main/kotlin/net/corda/blobinspector/BlobInspector.kt]: I agree, I'll leave it the way it is then.

**Pull request merged**